### DISCUSSION TITLE
Please consider adding support for Liground!

### DISCUSSION THREAD

**Discussion by pp1992m** [official-pikafish/Pikafish#39] [Ideas]
Please consider adding support for Liground!

I tried to modify some code (uci.h uci.cpp ucioption.cpp), but without success. I don't know how to program it. I tried to modify the code according to fairy-stockfish and multi-variant-stockfish. If you can modify the code so that Pikafish supports Liground, it would be a boon for MacOS and Linux users.

https://github.com/ml-research/liground
https://github.com/ianfab/Fairy-Stockfish/releases/fairy_sf_13
https://github.com/ddugovic/Stockfish/releases/variant_sf_10

**Comment by PikaCat-OuO** [comment]
For liground please use this by mtaktikos:
https://github.com/mtaktikos/Pikafish/releases

**Reply by pp1992m** [reply]
I see! Thank you!
